src/client.toit

```diff
@@ -74,6 +74,7 @@ class Client:
   close:
     // TODO(anders): This can block, fix me.
     incoming_.send null
+    send_ DisconnectPacket
```

Comment:
Can this fail? If so, we probably still want to cancel the task and set the pending things to null.

Reply:
added a try/finally.